.github/workflows/staging-build-tar.yml

```diff
@@ -459,3 +459,50 @@ jobs:
           working-directory: kibana/plugins/anomaly-detection-kibana-plugin
           command: yarn cy:run-with-security --config baseurl=http://localhost:5601
     
+  Test-Perf-Analyzer-No-Security:
+    runs-on: ubuntu-18.04
+    name: Test-Perf-Analyzer-No-Security
+    strategy:
+      matrix:
+        java: [12]
+    steps:
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: Set up AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Checkout opendistro-build
+        uses: actions/checkout@v1
+      - name: Checkout perf analyzer RCA package
+        uses: actions/checkout@v1
+        with:
+          repository: opendistro-for-elasticsearch/performance-analyzer-rca
+          # TODO: Change to a tag once new tags have been published
+          ref: master
+      - name: Checkout perf analyzer package
+        uses: actions/checkout@v1
+        with:
+          repository: opendistro-for-elasticsearch/performance-analyzer
+          # TODO: Change to a tag once new tags have been published
+          ref: master
+      - name: Build RCA into MavenLocal
+        run: |
+          cd ${GITHUB_WORKSPACE}/../performance-analyzer-rca
+          ./gradlew publishToMavenLocal
+      - name: Update SHA
+        run: |
+          cd ${GITHUB_WORKSPACE}/../performance-analyzer
+          rm licenses/performanceanalyzer-rca-1.3.jar.sha1
+          ./gradlew updateShas
+      - name: Run integration tests
+        run: |
+          .github/scripts/setup_runners_service.sh zip --es-nosec
+          export PATH=$JAVA_HOME:$PATH
+          cd ${GITHUB_WORKSPACE}/../performance-analyzer
+          pwd
+          ./gradlew integTest -Dtests.enableIT=true -Dtests.useDockerCluster=false -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9300
```

Comment:
Sid has mentioned in the issue on these two specific curl to do after starting ES:


If an ES cluster is running with ODFE 1.9+ with PA 1.9+, the following commands will prep the cluster (enable PA and RCA):

    curl localhost:9200/_opendistro/_performanceanalyzer/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}'
    curl localhost:9200/_opendistro/_performanceanalyzer/rca/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}'

Reply:
Yeah add those lines here https://github.com/opendistro-for-elasticsearch/opendistro-build/pull/354/files#diff-dcd3e1c66f2aa852aabbbd3739826be3R507 
I am assuming this is feature you are talking about in **Test Results** section in order to pass the test